### PR TITLE
remnote: init at 1.7.6

### PR DESCRIPTION
**Pull request by max-niederman**
###### Description of changes

[RemNote](https://www.remnote.com)

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/remnote/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  src = fetchurl {
+    url = "https://download.remnote.io/RemNote-${version}.AppImage";
+    sha256 = "sha256-yRUpLev/Fr3mOamkFgevArv2UoXgV4e6zlyv7FaQ4RM=";
+  };
+
+  meta = with lib; {
+    description = "A note-taking application focused on learning and productivity";
+    homepage = "https://remnote.com/";
+    license = licenses.unfree;
```
Comment (legendofmiracles) [inline pkgs/applications/misc/remnote/default.nix:15]:
```suggestion
    license = licenses.unfree;
    platforms = platforms.linux;
```
Appimage only runs on linux. Also missing maintainer

**Review by legendofmiracles — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/remnote/default.nix (hunk 1 above)
Comment (max-niederman) [reply pkgs/applications/misc/remnote/default.nix:16]: Interesting; I had assumed `appimageTools` set the platforms. Anyway, both issues are fixed now.

**Review by gbtb — APPROVED** [review]

**Review by gbtb — APPROVED** [review]

File: pkgs/applications/misc/remnote/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{ lib, fetchurl, appimageTools }:
+
+appimageTools.wrapType2 rec {
+  name = "remnote";
+  version = "1.7.6";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/remnote/default.nix:5]: This must be pname.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/remnote/default.nix [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{ lib, fetchurl, appimageTools }:
+
+appimageTools.wrapType2 rec {
+  name = "remnote";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/remnote/default.nix:4]:
```suggestion
  pname = "remnote";
```

**Pull request merged**